cmd/kyma/version/checker.go

```diff
@@ -0,0 +1,46 @@
+package version
```

Comment:
Probably not the fitting pkg for this function. Can we move it to somewhere internal?